src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Reindex/ReindexSingleResourceRequestHandler.cs

```diff
@@ -28,6 +28,8 @@ public class ReindexSingleResourceRequestHandler : IRequestHandler<ReindexSingle
         private readonly ISearchIndexer _searchIndexer;
         private readonly IResourceDeserializer _resourceDeserializer;
 
+        private const string HttpPostName = "POST";
```

Comment:
I think this is already defined: Microsoft.AspNetCore.Http.HttpMethods

Reply:
That's a good idea. But that would mean Core would have a dependency on AspNetCore, which we want to avoid (since this handler is in Core).